pegasus/test/test_tutorials.rb

```diff
@@ -26,4 +26,34 @@ def test_tutorials_contents_immutable
     tutorial[:image] = 'xyz'
     refute_equal Tutorials.new(:tutorials).contents('code.org').first[:image], tutorial[:image]
   end
+
+  def test_sort_by_popularity_code_org
+    code_org = "code.org"
+
+    assert Tutorials.sort_by_popularity?(code_org, "post-hoc")
+    assert Tutorials.sort_by_popularity?(code_org, "pre-hoc")
+    assert Tutorials.sort_by_popularity?(code_org, false)
```

Comment:
Could also add `soon-hoc` and `actual-hoc` to cover the whole cycle of HOC-modes.  ([See this doc](https://docs.google.com/document/d/1npRdcuOhOPGghHXkBZain_7ZvRrqvizAbwT1gwlAaK8/edit))